action.yml

```diff
@@ -1,13 +1,12 @@
-name: 'Hello World'
-description: 'Greet someone and record the time'
+name: 'Cancel workflow action'
+description: 'Cancel previous workflows'
 inputs:
-  who-to-greet:  # id of input
-    description: 'Who to greet'
+  workflow_id:  # id of input
+    description: 'The id of your workflow'
     required: true
-    default: 'World'
 outputs:
   time: # id of output
-    description: 'The time we greeted you'
+    description: 'Ok'
```

Comment:
You can remove outputs since there are none.

Reply:
Agree 💯